### PR TITLE
scripts/drtprod: combine create of ua1/ua2 and PCR setup

### PR DESCRIPTION
**Pull request by dt**

### CODE REVIEW

File: scripts/drtprod [hunk 1]
```diff
@@ -308,7 +283,7 @@ EOF"
         # setup dns
         $0 dns drt-chaos create
         ;;
-      "drt-ua1")
+      "ua")
```
Comment (itsbilal) [inline scripts/drtprod:286]: nit: should this be `drt-ua` ?

**Review by itsbilal — APPROVED** [review]
LGTM

File: scripts/drtprod (hunk 1 above)
Comment (dt) [reply scripts/drtprod:286]: shrug. It isn't a cluster name anymore, since it makes both clusters, so I'm not sure it matters?

Comment (itsbilal) [reply scripts/drtprod:286]: Sounds good. Merge away

**Comment by dt** [comment]
TFTR!

bors r+

**Pull request merged**